src/components/CreateVmWizard/steps/BasicSettings.js

```diff
@@ -258,6 +258,10 @@ class BasicSettings extends React.Component {
   render () {
     const idPrefix = this.props.id || 'create-vm-wizard-basic'
     const data = this.props.data
+    const nameValidation = this.validateVmName()
```

Comment:
It's not clear to me why we need this change. Ben, could you, please, explain? Thanks :)

Reply:
I changed it a little bit.
Is it still not clear?